src/main/scala/com/microsoft/ml/spark/cognitive/AnamolyDetection.scala

```diff
@@ -196,6 +200,7 @@ class SimpleDetectAnomalies(override val uid: String) extends AnomalyDetectorBas
   }
 
   override def transform(dataset: Dataset[_]): DataFrame = {
+    logTransform(uid)
```

Comment:
curious if it would be good to pass the dataset to logTransform to enable richer logging (e.g. colums names, row counts,...)

Reply:
This sounds good! I'll look into that.